OptimizelySDK/OptlyConfig/OptimizelyConfigService.cs

```diff
@@ -227,8 +227,8 @@ private OptimizelyAttribute[] GetAttributes(ProjectConfig projectConfig)
             foreach (var featureFlag in projectConfig.FeatureFlags)   
             {
 
-                var featureExperimentMap = experimentsMapById.Where(expMap => featureFlag.ExperimentIds.Contains(expMap.Key))
-                    .ToDictionary(k => k.Value.Key, v => v.Value);
+                var featureExperimentMap = featureFlag.ExperimentIds.Select(experimentId => experimentsMapById[experimentId])
+                    .ToDictionary(experiment => experiment.Key, experiment => experiment);
```

Comment:
It looks like losing order here - it picks up experimentIds as an array, but sets to map immediately.

Reply:
yes i have run multiple times with unsorted scenario and works fine.